universal-application-tool-0.0.1/conf/messages

```diff
@@ -53,14 +53,16 @@ content.description=CiviForm lets you apply for many benefits at once by reusing
 #----------------------------------------------------------#
 
 # Address question labels and placeholder text
-label.street=Street address
-placeholder.street=Enter your street address
+label.street=Address line 1
+placeholder.street=Street address
+label.line2=Address line 2
```

Comment:
(nit) idk what the standard here is. Could we add `address` in here somewhere so its really obvious what line2 refers to?

Reply:
That's a good point - it's really unclear as is. Will improve.